exo-run/features/steps/steps-when.ls

```diff
@@ -29,6 +29,15 @@ module.exports = ->
       ..on 'ended', -> done!
 
 
+  @When /^running a "([^"]*)" application$/ timeout: 600_000, (@app-name, done) ->
+    @checkout-app @app-name
+    @app-dir := path.join process.cwd!, 'tmp', @app-name
+    @setup-app @app-dir, ~>
+      command = \exo-run
+      if process.platform is 'win32' then command += '.cmd'
+      @run-app {command}, done
```

Comment:
The exit code should actually be non-zero here. But I don't think I should check for an error code at all since the step is "When trying to run the '...' application" (after I address the comment above that is).